vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -2532,9 +2532,25 @@ public void deselect(T item) {
      * @see GridSelectionModel
      */
     public void deselectAll() {
+        setDeselectAllowed(this,true);
```

Comment:
Space is missing after comma

Reply:
After calling this method, the grid will allow deselection in the UI, even if it was not the case previously.